asapdiscovery-docking/asapdiscovery/docking/docking.py

```diff
@@ -71,7 +71,6 @@ def run_docking_oe(
     from asapdiscovery.data.openeye import oechem, oedocking
     from asapdiscovery.docking.analysis import calculate_rmsd_openeye
 
-    logger.info(oechem.OEThrow.OEErrorHandlerImplBase)
```

Comment:
yep so sorry about this, I was testing various options for logging and this just slipped through review into main